content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,218 @@
+
+Istiod is considered a highly privileged component, similar to that of the Kubernetes API server itself.
+* It has high Kubernetes RBAC privileges, typically including `Secret` read access and webhook write access.
+* When acting as the CA, it can provision arbitrary certificates.
+* When acting as the XDS control plane, it can program proxies to perform arbitrary behavior.
+
+As such, the security of the cluster is tightly coupled to the security Istiod.
+Following [Kubernetes security best practices](https://kubernetes.io/docs/concepts/security/) around Istiod access is paramount.
+
+### Istio CNI plugin
+
+Istio can optionally be deployed with the [Istio CNI Plugin DaemonSet](docs/setup/additional-setup/cni/).
+This DaemonSet is responsible for setting up networking rules in Istio to ensure traffic is transparently redirected as needed.
+This is an alternative to the `istio-init` container discussed [below](#sidecar-proxies).
+
+Because the CNI DaemonSet modifies networking rules on the node, it requires an elevated `securityContext`.
+However, unlike [Istiod](#istiod), this is a **node-local** privilege.
+The implications of this are discussed [below](#node-compromise).
+
+### Sidecar Proxies
+
+Istio may [optionally](docs/overview/dataplane-modes/) deploy a sidecar proxy next to an application.
+
+The sidecar proxy needs the network to be programmed to direct all traffic through the proxy.
+This can be done with the [Istio CNI plugin](#istio-cni-plugin) or by deploying an `initContainer` (`istio-init`) on the pod (this is done automatically if the CNI plugin is not deployed).
+The `istio-init` container requires `NET_ADMIN` and `NET_RAW` capabilities.
+However, these capabilities are only present during the initialization - the primary sidecar container is completely unprivileged.
+
+Additionally, the sidecar proxy does not have any associated Kubernetes RBAC privileges at all.
+
+Each sidecar proxy is authorized to request a certificate for the associated Pod Service Account. 
+
+### Gateways and Waypoints
+
+{{< gloss "gateway" >}}Gateways{{< /gloss >}} and {{< gloss "waypoint">}}Waypoints{{< /gloss >}} act as standalone proxy deployments.
+Unlike [sidecars](#sidecar-proxies), they do not require any networking modifications, and thus don't require any privilege.
+
+### Ztunnel
+
+{{< gloss "ztunnel" >}}Ztunnel{{< /gloss >}} acts as a node-level proxy.
+This task requires the `NET_ADMIN`, `SYS_ADMIN`, and `NET_RAW` capabilities.
+Like the [Istio CNI Plugin](#istio-cni-plugin), these are **node-local** privileges only.
+The Ztunnel does not have any associated Kubernetes RBAC privileges.
+
+Ztunnel is authorized to request certificates for any Service Accounts of pods running on the same node.
+Similar to [kubelet](https://kubernetes.io/docs/reference/access-authn-authz/node/), this explicitly does not allow requesting arbitrary
+certificates.
+This, again, ensures these privileges are **node-local** only.
```

Comment:
The effect of the identity issuance limitations is a little orthogonal the linux privileges and this statement reads a little differently to where its used elsewhere. Not sure of a better way to sum things up however.